src/Exadel.CrazyPrice/Exadel.CrazyPrice.Common/Interfaces/IDiscountRepository.cs

```diff
@@ -11,36 +11,36 @@ public interface IDiscountRepository
     {
 
         /// <summary>
-        /// Gets Discount Programs by SearchCriteria.
+        /// Gets Discounts by SearchCriteria.
         /// </summary>
         /// <returns></returns>
         Task<List<DiscountResponse>> GetDiscountsAsync(SearchCriteria searchCriteria);
 
         /// <summary>
-        /// Gets a Discount Program by id.
+        /// Gets a Discount by id.
         /// </summary>
-        /// <param name="id">The id Discount Program.</param>
+        /// <param name="id">The id Discount.</param>
         /// <returns></returns>
         Task<DiscountResponse> GetDiscountAsync(Guid id);
 
         /// <summary>
-        /// Upserts a Discount Program.
+        /// Upserts a Discount.
         /// </summary>
         /// <param name="item"></param>
         /// <returns></returns>
         Task<UpsertDiscountRequest> UpsertDiscountAsync(UpsertDiscountRequest item);
 
         /// <summary>
-        /// Upserts a Discount Programs.
+        /// Upserts Discounts.
         /// </summary>
         /// <param name="items"></param>
         /// <returns></returns>
-        Task<UpsertDiscountRequest[]> UpsertDiscountAsync(UpsertDiscountRequest[] items);
+        Task<List<UpsertDiscountRequest>> UpsertDiscountAsync(UpsertDiscountRequest[] items);
 
         /// <summary>
-        /// Removes Discount Program by ids Discount Program.
+        /// Removes Discounts by ids.
```

Comment:
Do we allow mass delete? Use List<> instead of array

Reply:
Yes. [Done](https://github.com/yegorl/exadel_discounts_be/pull/7/commits/c2dd3b89f463c3c2de8a3dbd0b15dc00da6760e2)